crates/services/upgradable-executor/src/executor.rs

```diff
@@ -116,7 +116,7 @@ impl<S, R> Executor<S, R> {
     /// This constant is used along with the `version_check` test.
     /// To avoid automatic bumping during release, the constant uses `-` instead of `.`.
     #[cfg(test)]
-    pub const CRATE_VERSION: &'static str = "0-25-2";
+    pub const CRATE_VERSION: &'static str = "0-26-0";
```

Comment:
We will use `0.26.0` as a starting binary for the testnet, so we don't need to increase the version.